src/shared/logging/transports/loggly.ts

```diff
@@ -9,6 +12,19 @@ const logglyTransport: transportFunctionType = async (msg, level, _options) => {
   const platform = Platform.OS;
   const versionCode = APP_VERSION_CODE;
   const versionName = APP_VERSION_NAME;
+  let status = '';
+  let minutes = 'n/a';
+
+  const today = getCurrentDate();
+  const exposureStatusJson = await AsyncStorage.getItem(EXPOSURE_STATUS);
+
+  if (exposureStatusJson) {
```

Comment:
Maybe move all this inside the LOGGLY if statement

Reply:
good point